docs/pages/desktop-access/introduction.mdx

```diff
@@ -1,44 +1,72 @@
 ---
-title: Desktop Access
-description: Teleport desktop access introduction and resources.
+title: Manage Windows resources
+description: Demonstrates how you can manage access to Windows desktops with Teleport.
```

Comment:
I would use a complete sentence for the meta description, since this is what users would see when they find this page via Google.

Reply:
I believe this is a complete sentence. I haven't found these descriptions to follow any particular convention, so I used the verb - action pattern. If you have a specific conventions you think we should follow, we can add it to the style guide.

Reply:
I was thinking something with a subject and a verb (this is just a verb phrase). That said, I'm not sure whether a particular sentence structure is more effective at converting users from Google search results than any other. This is probably a matter of taste, so your suggestion is fine 👍 .